src/components/AlbumThumbLoading.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { ImageIcon } from "./Icons"
+function AlbumThumbLoading() {
+  const LoadingAlbums = Array(4).fill(
+    <div className="singleAlbum stacked">
```

Comment:
Better to split this into two components, it's more composable, readable and reusable:

```tsx
const AlbumThumbPlaceholder = () =>
  <div className="singleAlbum stacked">
    <div className="front">
      <div className="wrapper">
        <ImageIcon className="icon is-loading-img" />
      </div>
      <div className="card-content">
        <div className="animation-name"></div>
        <div className="animation-date"></div>
      </div>
    </div>
  </div>

const AlbumThumbLoading = (num: number) =>
  <div className="main-container">
    {[...Array(num).keys()].map(_ => <AlbumThumbPlaceholder />)}
  </div>
```